dbt/models/marts/teachers/dim_teachers.sql

```diff
@@ -1,54 +1,100 @@
+
+-- model: dim_teachers__v2
+-- scope: all teachers, no including co-teachers 
```

Comment:
Co teachers are included here, because they have a user account type teacher
```suggestion
-- scope: all teachers
```

Reply:
This code can be used to validate that co-teachers are included

```select 
si.section_id 
, si.teacher_id 
, si.is_section_owner 
, t.teacher_id 
from dbt_natalia.stg_dashboard__section_instructors si
left join dbt_natalia.dim_teachers t on si.teacher_id = t.teacher_id
where si.section_id in (
4410895
, 4476680
, 3785088
, 4769654
, 4775209
, 4790631)
order by si.section_id , si.teacher_id ;```